File: lib/models/bukti_kompen.dart
class BuktiKompen {
  final int idBukti;
  final int idPengajuan;
  final String filePath;

  BuktiKompen({
    required this.idBukti,
    required this.idPengajuan,
    required this.filePath,
  });

  factory BuktiKompen.fromJson(Map<String, dynamic> json) {
    return BuktiKompen(
      idBukti: json['id_bukti'],
      idPengajuan: json['id_pengajuan'],
      filePath: json['file_path'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'id_bukti': idBukti,
      'id_pengajuan': idPengajuan,
      'file_path': filePath,
    };
  }
}

File: e-kompen-api/app/Models/BuktiKompen.php
<?php
namespace App\Models;
use Illuminate\Database\Eloquent\Model;

class BuktiKompen extends Model {
    protected $table      = 'bukti_kompen';
    protected $primaryKey = 'id_bukti';
    public $timestamps    = false;
    protected $fillable   = [
        'id_pengajuan', 'file_path'
    ];

    public function pengajuanKompen() {
        return $this->belongsTo(PengajuanKompen::class, 'id_pengajuan', 'id_pengajuan');
    }
}